app/views/session_timeout/_ping.js.erb

```diff
@@ -28,41 +28,13 @@ function success(data) {
 
   if (show_warning & !el) {
     cntnr.insertAdjacentHTML('afterbegin', warning_info);
-    initTimer(warning);
+    LoginGov.countdownTimer('#countdown', warning);
```

Comment:
so much better! as the last person who touched this js, I really appreciate the refactor